src/lib/server/models/discord/attachment/index.ts

```diff
@@ -0,0 +1,17 @@
+import { type InferOutput, nullable, number, object, optional, string } from 'valibot';
+import { Snowflake } from '$lib/server/models/discord/snowflake';
+
+export const Attachment = object({
+    id: optional(Snowflake),
+    filename: string(),
+    title: optional(string()),
+    description: optional(string()),
+    content_type: nullable(optional(string())),
+    size: optional(number()),
+    url: string(),
+    proxy_url: optional(string()),
```

Comment:
This is _surprisingly_ a required field according to the docs, but is that so in practice?
```suggestion
    proxy_url: string(),
```

Reply:
I'm not actually sure, but I'll skew toward the specification.